cfn-resources/cluster/mongodb-atlas-cluster.json

```diff
@@ -82,7 +82,7 @@
       "type": "object",
       "properties": {
         "DiskIOPS": {
-          "type": "string",
+          "type": "integer",
```

Comment:
Is this change related or needed? While the change appears correct, this is a breaking change and will require updating CDK as well.

Reply:
This is true. Thank you for calling it out.
I don't see a reason for why it is a string. Both resources in the TF provider use Integer.
Although it is a breaking change, it will only impact users upgrading the CDK/CFN resource, and the fix should be simple.

Reply:
suggest leave as is for now, if customer ever asks about it we can revisit. rather then forcing parity between the tools, once something is out customers in that community may already be used to it working this way so would lean on developer/customer empathy here in avoiding breaking changes when possible.

Reply:
Reverted in be5fdfe361cfd5df527ca9418c4ccb3594c320c3 and re-implemented in conversion in d1c504df5ec4e0ca8dbaf3a4ac01014383114dab